Gemfile

```diff
@@ -5,7 +5,7 @@ require File.join(Bundler::Plugin.index.load_paths("bundler-inject")[0], "bundle
 
 gem 'acts_as_list',        '~> 1.0'
 gem 'faraday',             '>= 0.17.0'
-gem 'insights-api-common', '~> 3.8'
+gem 'insights-api-common', :git => "https://github.com/mkanoor/manageiq-api-common.git", :branch => "support_args"
```

Comment:
Do we have to use git? can we wait until next gem releases? There is no blocker issue that we have solve by switching to use a git branch.
We can mark the PR wip until the new gem is released.

Reply:
@bzwei That is going to get stored in the user_context, we just wanted to get level set with the manager-common-gem changes and then start using the graphql_args.